modules/learn/pages/clusters-and-availability/xdcr-with-scopes-and-collections.adoc

```diff
@@ -23,6 +23,7 @@ Between a keyspace on the source cluster and a keyspace on the target cluster, t
 The correct pairings of keyspaces on source and target must therefore be specified by the administrator.
 
 These forms of mapping are further described below.
+Note that whereas replication based on _implicit_ mapping is available in both Couchbase Server Enterprise and Community Edition, replication based on _explicit_ mapping is available only in Couchbase Server Enterprise Edition.
```

Comment:
and "Migration rules feature"

Reply:
I've modified this to: Note that whereas replication based on _implicit_ mapping is available in both Couchbase Server Enterprise and Community Edition, replication based on _explicit_ mapping and _migration rules_ is available only in Couchbase Server Enterprise Edition.